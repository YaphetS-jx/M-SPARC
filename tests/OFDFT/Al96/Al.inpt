# Test: Al #
CELL: 16 24 32
MESH_SPACING: 0.5
BC: P P P
EXCHANGE_CORRELATION: LDA_PZ

TOL_PSEUDOCHARGE: 1.00E-08

OFDFT_FLAG: 1
OFDFT_LAMBDA: 0.2
TOL_OFDFT: 1E-3