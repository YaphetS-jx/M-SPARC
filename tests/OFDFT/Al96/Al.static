***************************************************************************
                            Atom positions                                 
***************************************************************************
Cartesian coordinates of atoms (Bohr):
      0.0000000000       0.0000000000      24.0000000000
      4.0000000000       0.0000000000      28.0000000000
      0.0000000000       4.0000000000      28.0000000000
      0.0000000000       0.0000000000      16.0000000000
      4.0000000000       0.0000000000      20.0000000000
      8.0000000000       0.0000000000      24.0000000000
     12.0000000000       0.0000000000      28.0000000000
      0.0000000000       4.0000000000      20.0000000000
      4.0000000000       4.0000000000      24.0000000000
      8.0000000000       4.0000000000      28.0000000000
      0.0000000000       8.0000000000      24.0000000000
      4.0000000000       8.0000000000      28.0000000000
      0.0000000000      12.0000000000      28.0000000000
      0.0000000000       0.0000000000       8.0000000000
      4.0000000000       0.0000000000      12.0000000000
      8.0000000000       0.0000000000      16.0000000000
     12.0000000000       0.0000000000      20.0000000000
      0.0000000000       4.0000000000      12.0000000000
      4.0000000000       4.0000000000      16.0000000000
      8.0000000000       4.0000000000      20.0000000000
     12.0000000000       4.0000000000      24.0000000000
      0.0000000000       8.0000000000      16.0000000000
      4.0000000000       8.0000000000      20.0000000000
      8.0000000000       8.0000000000      24.0000000000
     12.0000000000       8.0000000000      28.0000000000
      0.0000000000      12.0000000000      20.0000000000
      4.0000000000      12.0000000000      24.0000000000
      8.0000000000      12.0000000000      28.0000000000
      0.0000000000      16.0000000000      24.0000000000
      4.0000000000      16.0000000000      28.0000000000
      0.0000000000      20.0000000000      28.0000000000
      0.0000000000       0.0000000000       0.0000000000
      4.0000000000       0.0000000000       4.0000000000
      8.0000000000       0.0000000000       8.0000000000
     12.0000000000       0.0000000000      12.0000000000
      0.0000000000       4.0000000000       4.0000000000
      4.0000000000       4.0000000000       8.0000000000
      8.0000000000       4.0000000000      12.0000000000
     12.0000000000       4.0000000000      16.0000000000
      0.0000000000       8.0000000000       8.0000000000
      4.0000000000       8.0000000000      12.0000000000
      8.0000000000       8.0000000000      16.0000000000
     12.0000000000       8.0000000000      20.0000000000
      0.0000000000      12.0000000000      12.0000000000
      4.0000000000      12.0000000000      16.0000000000
      8.0000000000      12.0000000000      20.0000000000
     12.0000000000      12.0000000000      24.0000000000
      0.0000000000      16.0000000000      16.0000000000
      4.0000000000      16.0000000000      20.0000000000
      8.0000000000      16.0000000000      24.0000000000
     12.0000000000      16.0000000000      28.0000000000
      0.0000000000      20.0000000000      20.0000000000
      4.0000000000      20.0000000000      24.0000000000
      8.0000000000      20.0000000000      28.0000000000
      8.0000000000       0.0000000000       0.0000000000
     12.0000000000       0.0000000000       4.0000000000
      4.0000000000       4.0000000000       0.0000000000
      8.0000000000       4.0000000000       4.0000000000
     12.0000000000       4.0000000000       8.0000000000
      0.0000000000       8.0000000000       0.0000000000
      4.0000000000       8.0000000000       4.0000000000
      8.0000000000       8.0000000000       8.0000000000
     12.0000000000       8.0000000000      12.0000000000
      0.0000000000      12.0000000000       4.0000000000
      4.0000000000      12.0000000000       8.0000000000
      8.0000000000      12.0000000000      12.0000000000
     12.0000000000      12.0000000000      16.0000000000
      0.0000000000      16.0000000000       8.0000000000
      4.0000000000      16.0000000000      12.0000000000
      8.0000000000      16.0000000000      16.0000000000
     12.0000000000      16.0000000000      20.0000000000
      0.0000000000      20.0000000000      12.0000000000
      4.0000000000      20.0000000000      16.0000000000
      8.0000000000      20.0000000000      20.0000000000
     12.0000000000      20.0000000000      24.0000000000
     12.0000000000       4.0000000000       0.0000000000
      8.0000000000       8.0000000000       0.0000000000
     12.0000000000       8.0000000000       4.0000000000
      4.0000000000      12.0000000000       0.0000000000
      8.0000000000      12.0000000000       4.0000000000
     12.0000000000      12.0000000000       8.0000000000
      0.0000000000      16.0000000000       0.0000000000
      4.0000000000      16.0000000000       4.0000000000
      8.0000000000      16.0000000000       8.0000000000
     12.0000000000      16.0000000000      12.0000000000
      0.0000000000      20.0000000000       4.0000000000
      4.0000000000      20.0000000000       8.0000000000
      8.0000000000      20.0000000000      12.0000000000
     12.0000000000      20.0000000000      16.0000000000
     12.0000000000      12.0000000000       0.0000000000
      8.0000000000      16.0000000000       0.0000000000
     12.0000000000      16.0000000000       4.0000000000
      4.0000000000      20.0000000000       0.0000000000
      8.0000000000      20.0000000000       4.0000000000
     12.0000000000      20.0000000000       8.0000000000
     12.0000000000      20.0000000000       0.0000000000
Atomic forces (Ha/Bohr):
     -0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000      -0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
     -0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
      0.0000000000       0.0000000000       0.0000000000
     -0.0000000000       0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000       0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
     -0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
     -0.0000000000       0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000       0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000       0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000      -0.0000000000
      0.0000000000       0.0000000000       0.0000000000
      0.0000000000       0.0000000000       0.0000000000
     -0.0000000000       0.0000000000       0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
      0.0000000000      -0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000      -0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
      0.0000000000      -0.0000000000      -0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000       0.0000000000
     -0.0000000000      -0.0000000000      -0.0000000000
