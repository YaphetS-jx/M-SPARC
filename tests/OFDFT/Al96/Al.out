***************************************************************************
*                      M-SPARC v1.0.0 (Mar 24, 2021)                      *
*   Copyright (c) 2019 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                Date: 29-Mar-2021  Start time: 13:52:57                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 16.000000 24.000000 32.000000 
FD_GRID: 32 48 64
FD_ORDER: 6
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
EXCHANGE_CORRELATION: LDA_PZ
OFDFT_FLAG: 1
TOL_OFDFT: 1.00E-06
CALC_STRESS: 0
CALC_PRES: 0
TOL_PSEUDOCHARGE: 1.00E-08
REFERENCE_CUTOFF: 0.50
FIX_RAND: 0
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_DENSITY: 0
OUTPUT_FILE: ../tests/OFDFT/Al96/Al.out_1
***************************************************************************
                             Initialization                                
***************************************************************************
Mesh spacing                       :  0.500000 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  ../tests/OFDFT/Al96/Al.out_1
Total number of atom types         :  1
Total number of atoms              :  96
Total number of electrons          :  288
Atom type 1  (valence electrons)   :  Al 3
Pseudopotential                    :  Al_goodwin.psp8
lloc                               :  0
Pseudocharge radii of atom type 1  :  10.00 10.00 10.00
Number of atoms of type 1          :  96
Estimated total memory usage       :  683.63 MB
=====================================================================
                Orbital-Free DFT NLCG (OFDFT-NLCG#1)                
=====================================================================
Iteration      Free Energy (Ha/atom)   NLCG Error        Timing (sec)
1              -2.2117957040E+00       1.064E-02         0.104
2              -2.2126455819E+00       6.804E-03         0.072
3              -2.2132128588E+00       8.746E-03         0.081
4              -2.2130923737E+00       2.625E-03         0.081
5              -2.2132219476E+00       1.851E-03         0.072
6              -2.2132692721E+00       2.911E-03         0.072
7              -2.2132505774E+00       1.208E-03         0.080
8              -2.2132668143E+00       4.703E-04         0.071
9              -2.2132714609E+00       8.284E-04         0.072
10             -2.2132706602E+00       2.808E-04         0.072
11             -2.2132716856E+00       9.364E-05         0.071
12             -2.2132719545E+00       2.151E-04         0.071
13             -2.2132718887E+00       8.331E-05         0.081
14             -2.2132719681E+00       2.288E-05         0.071
15             -2.2132719885E+00       5.712E-05         0.071
16             -2.2132719843E+00       2.098E-05         0.082
17             -2.2132719895E+00       4.780E-06         0.072
18             -2.2132719902E+00       7.770E-06         0.091
19             -2.2132719902E+00       2.052E-06         0.071
20             -2.2132719903E+00       1.894E-06         0.071
21             -2.2132719903E+00       3.379E-06         0.071
22             -2.2132719903E+00       1.302E-06         0.083
23             -2.2132719903E+00       8.030E-07         0.071
====================================================================
                                Energy                              
====================================================================
Free energy per atom               : -2.2132719903E+00 (Ha/atom)
Total free energy                  : -2.1247411107E+02 (Ha)
Kinetic energy                     :  7.9306540993E+01 (Ha)
Exchange correlation energy        : -7.5495677957E+01 (Ha)
Self and correction energy         : -3.9316341390E+02 (Ha)
Average force                      :  7.3619218234E-10 (Ha/Bohr)
Maximum force                      :  1.0590459345E-09 (Ha/Bohr)
Time for force calculation         :  4.067 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  8.713 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*                Main Developers: Qimen Xu, Abhiraj Sharma                *
*     Acknowledgements: U.S. DOE (DE-SC0019410); NSF (1333500,1553212)    *
***************************************************************************
                                                                           
